src/routes/api/trips.routes.js

```diff
@@ -136,5 +136,5 @@ router.post('/request', [checkUser, dateValidator, tripRequestValidator], tripsC
  *         description: Travel request successfully created
  */
 router.post('/multi-city', validateMultiCity, checkUser, tripsController.createMultiCityTrip);
-
+router.get('/request', checkUser, tripsController.getTrips);
```

Comment:
```suggestion
router.get('/', checkUser, tripsController.getTrips);
```
Just a suggestion, I know in the creation of a one-way trip request you are using ```/request``` but if you change  GET request to ```/api/v1/trips``` that would look nice.